06.bowling_object/lib/game.rb

```diff
@@ -20,9 +19,10 @@ def parse_marks(marks_text)
     shots = marks.map { |mark| Shot.new(mark) }
     Array.new(10) do |idx|
       if idx == 9
-        shots
+        Frame.new(idx, shots)
       else
-        shots.first.strike? ? shots.shift(1) : shots.shift(2)
+        shots_in_a_frame = shots.first.strike? ? shots.shift(1) : shots.shift(2)
```

Comment:
```suggestion
        shots_per_frame = shots.first.strike? ? shots.shift(1) : shots.shift(2)
```

per=〜ごとの、という意味があるので、「フレームごとのshots」みたいな命名にできます。
（変数やメソッド名にa/anが入ることはまれなので）